glm/ext/quaternion_common.inl

```diff
@@ -104,7 +104,7 @@ namespace glm
         {
             // Graphics Gems III, page 96
             T angle = acos(cosTheta);
-            T phi = angle + k * glm::pi<T>();
+            T phi = angle + static_cast<float>(k) * glm::pi<T>();
```

Comment:
It should be T here right?

Reply:
@christophe-lunarg you are right, I have updated the pull with this fixed